graylog2-server/src/test/java/org/graylog/testing/containermatrix/ContainerMatrixTestEngine.java

```diff
@@ -80,18 +81,33 @@ private Set<Class<? extends PluginJarsProvider>> getPluginJarsProvider(Set<Class
         return get(annotatedClasses, (ContainerMatrixTestsConfiguration annotation) -> Stream.of(annotation.pluginJarsProvider()));
     }
 
+    private boolean isJenkinsOnGithub() {
```

Comment:
How about renaming the method to something like `isRunningInCI` or something similar?

Reply:
not visible on this screen, but I renamed it to the suggested name